dreem/training/configs/override.yaml

```
@@ -0,0 +1,142 @@
+optimizer:
+  lr: 0.0001
+  weight_decay: 0
+
+scheduler:
+  factor: 0.5
+  patience: 5
+  threshold: 0.001
+
+dataset:
+  train_dataset:
+    dir: 
```

yamllint

[error] trailing spaces

(trailing-spaces)